tls/s2n_security_policies.h

```diff
@@ -24,14 +24,14 @@
 /* Kept up-to-date by s2n_security_policies_test */
 #define NUM_RSA_PSS_SCHEMES 6
 
-struct s2n_security_policy {
+typedef struct s2n_security_policy {
     uint8_t minimum_protocol_version;
     const struct s2n_cipher_preferences *cipher_preferences;
     const struct s2n_kem_preferences *kem_preferences;
     const struct s2n_signature_preferences *signature_preferences;
     const struct s2n_signature_preferences *certificate_signature_preferences;
     const struct s2n_ecc_preferences *ecc_preferences;
-};
+} s2n_security_policy;
```

Comment:
It's not necessary, but it makes it easier to work with. This way, I can omit the `struct` keyword when I declare this

Reply:
Does it make it easier to work with? You can leave off the word "struct", but that arguably loses some information. Most s2n structs also aren't typdefed, so this would be the exception rather than the norm. Let's stick to existing code base conventions.

Also, I only found one place where you actually left off the "struct", and many places where you still included it :)

Reply:
Ok. Will change